plugins/native/module/vtkF3DQuakeMDLImporter.h

```diff
@@ -10,6 +10,7 @@
 #ifndef vtkF3DQuakeMDLImporter_h
 #define vtkF3DQuakeMDLImporter_h
 
+#include <memory>
```

Comment:
Please put this include after `<vtkF3DImporter.h>` (with an empty line to split in two groups)